src/module/auth/login-form/FormLayout.tsx

```diff
@@ -0,0 +1,88 @@
+'use client';
+
+import { useEffect, useState } from 'react';
+import { useForm, FormProvider } from 'react-hook-form';
+import { Alert } from 'react-bootstrap';
+import Link from 'next/link';
+import { AccountButton, LoginForm } from '@/shared/components/account/AccountElements';
+import { AUTH_TOKEN, EMAIL, REMEMBER_ME } from '@/shared/constants/storage';
+import { useRouter } from 'next/navigation';
+import { useSearchParams } from '@/hooks/useSearchParams';
+import { useMutation } from '@apollo/client';
+import { USER_LOGIN } from '@/graphql/auth';
+import LoginFormGroup from './LoginFormGroup';
+
+type loginData = { email: string; password: string; remember_me: boolean };
+
+const FormLayout = () => {
+  const methods = useForm({
+    defaultValues: {
+      email: '',
+      password: '',
+      remember_me: false,
+    },
+  });
+  const { handleSubmit, watch } = methods;
+
+  const rememberMe = watch('remember_me');
+
+  useEffect(() => {
+    if (rememberMe !== undefined && typeof window !== 'undefined') {
+      localStorage.setItem(REMEMBER_ME, rememberMe.toString());
```

Comment:
Why not do this when form submit? Seems a little bit off to have this as an effort

Reply:
This is merged from dev= = really don't konw

Reply:
okie, let's keep it here for now